examples/android/binder/java/io/grpc/binder/cpp/exampleserver/BUILD

```diff
@@ -26,8 +26,7 @@ cc_library(
     ],
     deps = [
         # Temporarily directly depend on this target before we expose a public API
-        # TODO(mingcl): Uncomment this after server interfaces are merged
-        # "//src/core/ext/transport/binder/server:grpc_transport_binder_server",
+        "//src/core/ext/transport/binder/server:grpc_transport_binder_server",
```

Comment:
I'm not sure if there's such a rule, but should `//:grpc` goes before `//src/...`?

Reply:
For some reason my buildifier (formatter) does not care the order here.. (but it does care about indent).

I guess we can just keep it as is, we will need to touch it again later anyway after we have public API